namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-preprod/03-resourcequota.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: namespace-quota
+  namespace: hmpps-book-secure-move-api-preprod
+spec:
+  hard:
+    requests.cpu: 500m
```

Comment:
Could you use the same `pods: "50"` limit here as in staging?

This PR is going to be merged in a day or so, which will change the production namespace resourcequota, so it would be best to stay consistent with that.

https://github.com/ministryofjustice/cloud-platform-environments/pull/1538

Reply:
:+1: done